clash-lib/prims/commonverilog/Clash_Xilinx_ClockGen.primitives.yaml

```diff
@@ -4,39 +4,51 @@
     type: |-
       clockWizard
         :: ( KnownDomain domIn confIn       -- ARG[0]
-           , KnownDomain domOut confOut )   -- ARG[1]
-        => SSymbol name                    -- ARG[2]
-        -> Clock  pllIn                    -- ARG[3]
-        -> Reset pllIn                     -- ARG[4]
-        -> (Clock pllOut, Enable pllOut)
+           , KnownDomain domOut confOut     -- ARG[1]
+           , KnownDomain domPllLock confPllLock ) -- ARG[2]
```

Comment:
`KnownDomain` takes one argument; could you fix that?

Reply:
Please also look at the layout: the comments should be vertically aligned. This might "automatically" happen by fixing `KnownDomain` though ;-).